### ISSUE TITLE
"My Notes" page is confusing when there are no notes.

### ISSUE THREAD

**Issue by drumrock** [scrimba/community#516]
If I have no notes, "My Notes" page still shows some UI elements which make sense only with existing notes.

They can be hidden until user has any notes.

And some clear statement may be presented, like "You don't have any notes yet.".

![image](https://user-images.githubusercontent.com/10419376/87622143-21a1e280-c72b-11ea-8db1-5c025d8c108b.png)

**Comment by scanf** [comment]
Agree, we should have a default view when you are there for the first time. I will address this together with some other UI things, thanks 🙏
